Add Grafana dashboard for Plus (#1194)
nginxinc/kubernetes-ingress: docs-web/logging-and-monitoring/grafana/README.md

## Prerequisites

The dashboard has been tested with the following software versions:

* Grafana >= v7.2.0
* Prometheus >= v2.18.1

A Prometheus data source needs to be [added](https://prometheus.io/docs/visualization/grafana/#using) before installing the dashboard.

## Installing the Dashboard

In the Grafana UI complete the following steps:

1. Use the *New Dashboard* button and click *Import*.
2. Upload `NGINXPlusICDashboard.json` or copy and paste the contents of the file in the textbox and click *Load*.
3. Set the Prometheus data source and click *Import*.
4. The dashboard will appear. Note how you can filter the instance label just below the dashboard title (top left corner). This allows you to filter metrics per instance. By default, all instances are selected.

![dashboard](./dashboard.png)

## Graphs

The dashboard comes with 4 rows with the following graphs for NGINX metrics:

* Environment Variables
    * Nginx Plus Reload (`nginx_last_reload_status`). This graph shows the state of the last Nginx Plus reload, `Successful`/`Failed`.
    * Last Reload Time (`nginx_last_reload_milliseconds`)graph, shows duration of the last reload in milliseconds.

>     * Reloads ( `nginx_reloads_total`). This graph shows the total times Nginx has reloaded.

Reviewer: ```suggestion
    * Reloads (`nginx_reloads_total`). This graph shows the total times Nginx has reloaded.
```

    * Reload Errors (`nginx_reload_errors_total`) graph shows the total number of times Nginx has failed to reload.

>     * Netowrk I/O (`nginxplus_server_zone_received` and `nginxplus_server_zone_sent`). This graphs shows the traffic sent and recieved by the Ingress Controller.

Reviewer: ```suggestion
    * Network I/O (`nginxplus_server_zone_received` and `nginxplus_server_zone_sent`). This graphs shows the traffic sent and received by the Ingress Controller.
```

* Ingress Metrics

>     * HTTP Request Volume (`nginxplus_http_requests_total`). This graphs shows the total number of HTTP requests recieved by the Ingress Controller.

Reviewer: ```suggestion
    * HTTP Request Volume (`nginxplus_http_requests_total`). This graphs shows the total number of HTTP requests received by the Ingress Controller.
```
